### ISSUE TITLE
Linux - Chrome password 

### ISSUE THREAD

**Issue by jboss38** [AlessandroZ/LaZagne#107]
Hello, 

Any chrome password reader plan ? 

Can you add it for Linux ?
Do you know how to do it ?

**Comment by AlessandroZ** [comment]
It has already been implemented on Linux. Chrome passwords are stored on the system wallet (gnome keyring, KDE wallet, etc.).

**Comment by jboss38** [comment]
Well it's not working then.

Chrome store password here :  `~/.config/google-chrome/Default/Login Data`

I try laZagne on linux with chrome and lot of password stored, nothing appear.

**Comment by AlessandroZ** [comment]
I have tested it, and it's still working for me. I use Gnome and Chrome passwords are printed on "Gnomekeyring passwords" category. So there should appear using "python laZagne.py wallet" and not the browser category. 
If you are on Gnome, you could use "Seahorse" to check if chrome passwords are stored on the system wallet. 
Right now, only Gnome and KDE wallet are supported. 

Which system are you using ?

**Comment by jboss38** [comment]
I tested on Linux Mint MATE edition.

I'm going to test on Ubuntu 15+

**Comment by jboss38** [comment]
Tested on Ubuntu 15, also not working.

See : http://nsa37.casimages.com/img/2017/02/23/170223121509995381.png

**Comment by AlessandroZ** [comment]
aaaah, it's because you launch it using the root account

**Comment by jboss38** [comment]
also not working with user account, tested

**Comment by jboss38** [comment]
![sss](https://cloud.githubusercontent.com/assets/6693379/23256620/822650b4-f9c0-11e6-849f-8811fbc0c20b.PNG)

**Comment by AlessandroZ** [comment]
I cannot debug your problem. I have tested on Mint and Ubuntu this morning and it worked. Be sure to launch it using the same user than the passwords are stored. 

Launch "seahorse" to see the content of the Gnome keyring and if it's empty, it's normal the password is not be shown.

**Comment by jboss38** [comment]
I can see the password here : Applications -> Accessories -> Password and Encryption Keys

![sss](https://cloud.githubusercontent.com/assets/6693379/23259360/0d30cdea-f9cd-11e6-913c-ebe0a784e07f.PNG)

I try to use seahorse but the wiki is mindfck & don't want spend to much time on this (seahorse).

But it's strange cause my ubuntu is a fresh install...
Did you use Ubuntu 16 ?

**Comment by SasiKiranK** [comment]
I have found the Login Data file in the profile folder. But it doesn;t have any login information.
Login table is empty.
I have deleted that file and checked in chrome, still, it can pull up the saved passwords.
I assume the latest version Chrome stores the password somewhere else.

I am using Ubuntu on and Chrome Version 57.0.2987.110 (64-bit)

Attached the screenshot for ref http://nimb.ws/rDp5Xm
